packages/core/src/sso/OktaSsoConnector/index.ts

```diff
@@ -0,0 +1,63 @@
+import { ConnectorError, ConnectorErrorCodes } from '@logto/connector-kit';
+import { SsoProviderName } from '@logto/schemas';
+import { conditional } from '@silverhand/essentials';
+import camelcaseKeys from 'camelcase-keys';
+
+import assertThat from '#src/utils/assert-that.js';
+
+import { fetchToken, getUserInfo, getIdTokenClaims } from '../OidcConnector/utils.js';
+import { OidcSsoConnector } from '../OidcSsoConnector/index.js';
+import { type SingleSignOnFactory } from '../index.js';
+import { basicOidcConnectorConfigGuard } from '../types/oidc.js';
+import { type ExtendedSocialUserInfo } from '../types/saml.js';
+import { type SingleSignOnConnectorSession } from '../types/session.js';
+
+export class OktaSsoConnector extends OidcSsoConnector {
+  /**
+   * Override the getUserInfo method from the OidcSsoConnector class
+   *
+   * @remark Okta's IdToken does not include the sufficient user claims like email_verified, phone_verified, etc. {@link https://devforum.okta.com/t/email-verified-claim/3516/2}
+   * This method will fetch the user info from the userinfo endpoint instead.
+   */
```

Comment:
Will there be sync issues between Okta and Logto userinfo endpoint?

Reply:
What do you mean by sync issues?

Reply:
Move forward for bug bash. Will offline sync.